### PR TITLE
Update to 1.0.1 and use pcre2 as the regex backend

### PR DESCRIPTION
**Pull request by mingwandroid**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 6a1fa16a7f6335ce8b2630fbdbb5e57c4027929ebc56fcd1ac55edb141b409b4
-  patches:
-    - htonll-osx.patch
```
Comment (xhochy) [inline recipe/meta.yaml:12]:
```suggestion
    - htonll-osx.patch
```
This patch is still required.

Comment (mingwandroid) [reply recipe/meta.yaml:12]: ok, I'll rebase it.

**Comment by xhochy** [comment]
Thanks @mingwandroid!

**Comment by mingwandroid** [comment]
Thank you!

**Pull request merged**